File: C++/1207.cpp
#include <iostream>
using namespace std;

int main() {
	int a, b, c, d;
	
	cin>>a;
	cin>>b;
	cin>>c;
	cin>>d;
	
	int sum = a + b + c + d;
	if (sum == 1) {
    	cout<<"도"<<endl;
	}
	else if (sum == 2) {
		cout<<"개"<<endl;
	}
	else if (sum == 3) {
	    cout<<"걸"<<endl;
	}
	else if (sum == 4) {
		cout<<"윷"<<endl;
	}
	else {
		cout<<"모"<<endl;
	}
	return 0;
}

File: Py/1207.py
a, b, c, d = map(int,input().split())
sum = a+b+c+d
if sum == 1:
    print("도")
elif sum == 2:
    print("개")
elif sum == 3:
    print("걸")
elif sum == 4:
    print("윷")
else:
    print("모")
